[2022-07-30 02:50:56.156] [remoteagent] [info] Extension host agent started.
[2022-07-30 02:50:57.781] [remoteagent] [info] [127.0.0.1][84c1a84a][ManagementConnection] New connection established.
[2022-07-30 02:50:57.782] [remoteagent] [info] [127.0.0.1][a36e96d5][ExtensionHostConnection] New connection established.
[2022-07-30 02:50:57.788] [remoteagent] [info] [127.0.0.1][a36e96d5][ExtensionHostConnection] <8673> Launched Extension Host Process.
[2022-07-30 02:51:51.037] [remoteagent] [info] [127.0.0.1][a36e96d5][ExtensionHostConnection] <8673> Extension Host Process exited with code: 0, signal: null.
[2022-07-30 02:51:51.037] [remoteagent] [info] Last EH closed, waiting before shutting down
[2022-07-30 02:51:56.000] [remoteagent] [info] Got delay-shutdown request while in shutdown timeout, delaying
[2022-07-30 02:51:56.001] [remoteagent] [info] Cancelling previous shutdown timeout
[2022-07-30 02:51:56.526] [remoteagent] [info] [127.0.0.1][3e9ef7cc][ManagementConnection] New connection established.
[2022-07-30 02:51:56.654] [remoteagent] [info] [127.0.0.1][821d044c][ExtensionHostConnection] New connection established.
[2022-07-30 02:51:56.662] [remoteagent] [info] [127.0.0.1][821d044c][ExtensionHostConnection] <8840> Launched Extension Host Process.
[2022-07-30 02:51:57.301] [remoteagent] [info] [127.0.0.1][84c1a84a][ManagementConnection] The client has disconnected, will wait for reconnection 3h before disposing...
[2022-07-30 02:51:57.310] [remoteagent] [error] Error: Unexpected SIGPIPE
    at process.<anonymous> (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/server/node/server.main.js:145:7933)
    at process.emit (node:events:390:28)
[2022-07-30 02:56:56.002] [remoteagent] [info] New EH opened, aborting shutdown
[2022-07-30 05:16:30.284] [remoteagent] [info] [127.0.0.1][84c1a84a][ManagementConnection] Another client has connected, will shorten the wait for reconnection 5m before disposing...
[2022-07-30 05:16:30.285] [remoteagent] [info] [127.0.0.1][821d044c][ExtensionHostConnection] The client has reconnected.
[2022-07-30 05:16:30.448] [remoteagent] [info] [127.0.0.1][3e9ef7cc][ManagementConnection] The client has reconnected.
[2022-07-30 05:16:31.540] [remoteagent] [error] ["ErrorNoTelemetry: Could not find pty on pty host","    at d._throwIfNoPty (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:29103)","    at d.updateIcon (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:25409)","    at Object.call (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:16:8293)","    at onPromise (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4880)","    at onRawMessage (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4268)","    at /home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:3556","    at h.invoke (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at z.deliver (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at M.fire (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.W (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:9:20668)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-07-30 05:16:31.540] [remoteagent] [error] ["ErrorNoTelemetry: Could not find pty on pty host","    at d._throwIfNoPty (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:29103)","    at d.updateTitle (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:25349)","    at Object.call (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:16:8293)","    at onPromise (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4880)","    at onRawMessage (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4268)","    at /home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:3556","    at h.invoke (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at z.deliver (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at M.fire (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.W (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:9:20668)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-07-30 05:16:31.541] [remoteagent] [error] ["ErrorNoTelemetry: Could not find pty on pty host","    at d._throwIfNoPty (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:29103)","    at d.updateTitle (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:25349)","    at Object.call (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:16:8293)","    at onPromise (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4880)","    at onRawMessage (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4268)","    at /home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:3556","    at h.invoke (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at z.deliver (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at M.fire (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.W (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:9:20668)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-07-30 05:16:31.541] [remoteagent] [error] ["ErrorNoTelemetry: Could not find pty on pty host","    at d._throwIfNoPty (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:29103)","    at d.updateIcon (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:25409)","    at Object.call (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:16:8293)","    at onPromise (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4880)","    at onRawMessage (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4268)","    at /home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:3556","    at h.invoke (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at z.deliver (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at M.fire (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.W (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:9:20668)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-07-30 05:16:31.541] [remoteagent] [error] ["ErrorNoTelemetry: Could not find pty on pty host","    at d._throwIfNoPty (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:29103)","    at d.updateTitle (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:25349)","    at Object.call (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:16:8293)","    at onPromise (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4880)","    at onRawMessage (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4268)","    at /home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:3556","    at h.invoke (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at z.deliver (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at M.fire (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.W (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:9:20668)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-07-30 05:16:31.542] [remoteagent] [error] ["ErrorNoTelemetry: Could not find pty on pty host","    at d._throwIfNoPty (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:29103)","    at d.updateTitle (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:25349)","    at Object.call (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:16:8293)","    at onPromise (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4880)","    at onRawMessage (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4268)","    at /home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:3556","    at h.invoke (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at z.deliver (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at M.fire (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.W (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:9:20668)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-07-30 05:21:30.452] [remoteagent] [info] [127.0.0.1][84c1a84a][ManagementConnection] The reconnection short grace time of 5m has expired, so the connection will be disposed.
[2022-07-30 06:47:24.172] [remoteagent] [info] [127.0.0.1][ca12a0af][ManagementConnection] New connection established.
[2022-07-30 06:47:24.272] [remoteagent] [info] [127.0.0.1][9ffbba10][ExtensionHostConnection] New connection established.
[2022-07-30 06:47:24.277] [remoteagent] [info] [127.0.0.1][9ffbba10][ExtensionHostConnection] <10886> Launched Extension Host Process.
[2022-07-30 06:47:25.770] [remoteagent] [info] [127.0.0.1][3e9ef7cc][ManagementConnection] The client has disconnected, will wait for reconnection 3h before disposing...
[2022-07-30 06:47:25.770] [remoteagent] [error] Error: Unexpected SIGPIPE
    at process.<anonymous> (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/server/node/server.main.js:145:7933)
    at process.emit (node:events:390:28)
[2022-07-30 06:47:26.766] [remoteagent] [error] ["ErrorNoTelemetry: Could not find pty on pty host","    at d._throwIfNoPty (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:29103)","    at d.updateTitle (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:25349)","    at Object.call (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:16:8293)","    at onPromise (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4880)","    at onRawMessage (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4268)","    at /home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:3556","    at h.invoke (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at z.deliver (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at M.fire (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.W (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:9:20668)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-07-30 06:48:35.216] [remoteagent] [info] [127.0.0.1][9ffbba10][ExtensionHostConnection] <10886> Extension Host Process exited with code: 0, signal: null.
[2022-07-30 06:48:39.777] [remoteagent] [info] [127.0.0.1][3e9ef7cc][ManagementConnection] Another client has connected, will shorten the wait for reconnection 5m before disposing...
[2022-07-30 06:48:39.777] [remoteagent] [info] [127.0.0.1][79c2ca8f][ManagementConnection] New connection established.
[2022-07-30 06:48:39.867] [remoteagent] [info] [127.0.0.1][36e67dfe][ExtensionHostConnection] New connection established.
[2022-07-30 06:48:39.872] [remoteagent] [info] [127.0.0.1][36e67dfe][ExtensionHostConnection] <11068> Launched Extension Host Process.
[2022-07-30 06:48:39.917] [remoteagent] [info] [127.0.0.1][ca12a0af][ManagementConnection] The client has disconnected, will wait for reconnection 3h before disposing...
[2022-07-30 06:48:39.918] [remoteagent] [error] Error: Unexpected SIGPIPE
    at process.<anonymous> (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/server/node/server.main.js:145:7933)
    at process.emit (node:events:390:28)
[2022-07-30 06:52:24.267] [remoteagent] [info] [127.0.0.1][821d044c][ExtensionHostConnection] <8840> Extension Host Process exited with code: 0, signal: null.
[2022-07-30 06:53:40.068] [remoteagent] [info] [127.0.0.1][3e9ef7cc][ManagementConnection] The reconnection short grace time of 5m has expired, so the connection will be disposed.
[2022-07-30 09:48:50.552] [remoteagent] [info] [127.0.0.1][ca12a0af][ManagementConnection] The reconnection grace time of 3h has expired, so the connection will be disposed.
[2022-07-30 10:21:29.704] [remoteagent] [info] [127.0.0.1][36e67dfe][ExtensionHostConnection] <11068> Extension Host Process exited with code: 0, signal: null.
[2022-07-30 10:21:29.704] [remoteagent] [info] Last EH closed, waiting before shutting down
[2022-07-30 10:26:29.705] [remoteagent] [info] Last EH closed, shutting down
